daemon/api.go

```diff
@@ -585,21 +585,50 @@ func deleteOp(c *Command, r *http.Request) Response {
 	}
 }
 
+type licT struct {
+	Intro   string
+	License string
+	Agreed  bool
+}
+
+func (*licT) Error() string {
+	// licT isn't an error, strictly speaking, but can
+	// be converted into one for convenience
```

Comment:
It's much cheaper to prevent the awkwardness here than it is document it.

Initially I thought of something like:

    type noLicenseError *licenseData

But actually, the error is not even using the data, so this is simply

    var noLicenseError = errors.New("license agreement required")